src/frontend/src/flows/manage/index.ts

```diff
@@ -14,25 +13,46 @@ import { settingsIcon } from "../../components/icons";
 import { displayError } from "../../components/displayError";
 import {
   authenticateBox,
-  AuthTemplates,
+  AuthnTemplates,
 } from "../../components/authenticateBox";
 import { setupRecovery } from "../recovery/setupRecovery";
+import { recoveryWizard } from "../recovery/recoveryWizard";
 import { hasOwnProperty } from "../../utils/utils";
 import { pollForTentativeDevice } from "../addDevice/manage/pollForTentativeDevice";
 import { chooseDeviceAddFlow } from "../addDevice/manage";
 import { addLocalDevice } from "../addDevice/manage/addLocalDevice";
 import { warnBox } from "../../components/warnBox";
 
 /* Template for the authbox when authenticating to II */
-export const authTemplatesManage: AuthTemplates = {
-  message: html`<p class="t-lead">Authenticate to manage your anchor</p>`,
-  button: "Authenticate",
+export const authnTemplateManage = (): AuthnTemplates => {
+  const wrap = (slot: string): TemplateResult => html`
+    <div class="t-centered l-stack">${slot}</div>
+  `;
+  return {
+    new: {
+      slot: wrap(`Anonymously connect to dapps on the Internet Computer`),
+      useExistingText: "Manage Existing",
+    },
+    useExisting: {
+      slot: wrap(`Enter your Anchor to continue to Internet Identity`),
+    },
+
+    pick: { slot: wrap("Choose an Anchor") },
+  };
 };
 
 /* the II authentication flow */
-export const authFlowManage = async (
-  connection: Connection
-): Promise<LoginData> => authenticateBox(connection, authTemplatesManage);
+export const authFlowManage = async (connection: Connection) => {
```

Comment:
This was moved from the top-level `index.ts` for more symmetry with the authz flow; also because it should have been here all along